DSN/src/main/resources/application.properties

```diff
@@ -9,9 +9,15 @@ spring.h2.console.enabled=true
 #spring.datasource.url=jdbc:mysql://localhost:3306/social_network?serverTimezone=UTC
 #spring.datasource.username=root
 #spring.datasource.password=password
-#spring.jpa.hibernate.ddl-auto=update
```

Comment:
Could you please revert back the changes which is not for the prometheus endpoing?